ui/raidboss/data/06-ew/raid/p8s.ts

```diff
@@ -753,31 +753,86 @@ const triggerSet: TriggerSet<Data> = {
       },
     },
     {
-      id: 'P8S Quadrupedal Impact',
+      id: 'P8S Quadrupedal Impact/Crush',
       type: 'StartsUsing',
-      netRegex: NetRegexes.startsUsing({ id: '7A04', source: 'Hephaistos', capture: false }),
-      alertText: (_data, _matches, output) => output.text!(),
-      outputStrings: {
-        text: {
-          en: 'Follow Jump',
-          de: 'Sprung folgen',
-          ja: '近づく',
-          ko: '보스 따라가기',
-        },
+      netRegex: NetRegexes.startsUsing({ id: ['7A04', '7A05'], source: 'Hephaistos' }),
+      delaySeconds: 0.5,
```

Comment:
Would you mind testing if this delay can go away with the new version of OverlayPlugin? If we don't have to delay this call, I think it would be better.

Reply:
Yea, I'll need to get a few pulls in to get the worst culprit which is something like boss facing north turns 275 degrees (lol).

Reply:
I only had a few pulls in tonight to test it with no delay, did not notice any issues.